# Manual curation of bird microchromosomes with HiC and gene mapping

#BGA24/sessions #GitPod #Curation

This session is part of [**Biodiversity Genomics Academy 2024**](https://thebgacademy.org)

[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/thebgacademy/bird-micro-curation) 

## Session Leader(s)

Genome Reference Informatics Team, Wellcome Sanger Institute

- Tom Mathers

## Description

By the end of this session you will be able to:

1. Understand the Manual curation pipeline in regards to Bird microchromosomes
2. Understand how Jbrowse can be utilised as a tool both to  aid curation and also for post curation analysis
3. Interpret a HiC Map
4. Understand how to curate an assembly using PretextView and associated curation scripts
5. Create an updated/curated assembly fasta

## Prerequisites

1. Understanding the terms genome assembly, reads, contigs
2. Understanding of linux command line basics
3. To have read/watched the rapid curation documentation available at <https://gitlab.com/wtsi-grit/rapid-curation/-/tree/main>
4. To have read through the Interpreting_HiC_maps guide
5. Have access to a 3 button mouse

>[!warning] "Please make sure you MEET THE PREREQUISITES and READ THE DESCRIPTION above"
>
>    You will get the most out of this session if you meet the prerequisites above.
>
>    Please also read the description carefully to see if this session is relevant to you.
>    
>    If you don't meet the prerequisites or change your mind based on the description or are no longer available at the session time, please email damon at thebgacademy.org to cancel your slot so that someone else on the waitlist might attend.


## Instructions


### Step 8. Normally after this one should re-map the HiC data to the new fasta file and check no errors were made or anything was missed in the first curation effort. We unfortunately do not have time for this during this practical. 
